frontend/www/js/omegaup/components/contest/NewForm.vue

```diff
@@ -320,14 +355,39 @@ export default class NewForm extends Vue {
     this.penalty = 20;
     this.penaltyType = 'none';
     this.showScoreboardAfter = true;
+    this.contestStyleDesc = T.contestNewFormTitlePlaceholderConacupStyle;
   }
 
   onSubmit() {
-    this.$emit('emit-update-contest', this);
-  }
-
-  mounted() {
-    $('.selectpicker', this.$el).selectpicker();
+    if (this.update) {
+      this.$emit('emit-update-contest', this);
+      return;
+    }
+    this.$emit('create-contest', {
```

Comment:
posiblemente sea mejor _en este caso_ separar la declaración del parámetro:

```suggestion
    const contest: omegaup.Contest = {
       ...
    };
    this.$emit('create-contest', contest);
```

la razón es porque así TypeScript se encarga de que el tipo es el esperado del otro lado. No supe cómo decirle a TypeScript "por favor checa que esta expresión que no está ligada a una variable tenga el tipo correcto", porque las opciones que hacen eso de manera concisa son [type assertions](https://www.typescriptlang.org/docs/handbook/basic-types.html#type-assertions), que hacen que el compilador confíe ciegamente en lo que se le está pasando.

Reply:
Listo. sirvió para arreglar otras cosas que se estaban yendo en ese request inline